### PR TITLE
 Added supporting for WS2808 LED driver

### PR DESCRIPTION
**Pull request by DemoS76**

### CODE REVIEW

File: src/devices/Ws28xx/BitmapImageWs2808.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        
+        public BitmapImageWs2808(int width, int height)
+            : base(new byte[width * height * BytesPerPixel], width, height, width * BytesPerPixel)
+        {
+        }
+
+        public override void SetPixel(int x, int y, Color c)
+        {
+            var offset = y * Stride + x * BytesPerPixel;
+            Data[offset++] = (byte)(c.G & 0xFF);
```
Comment (krwq) [inline src/devices/Ws28xx/BitmapImageWs2808.cs]: c.G is already a byte, no need to mask or cast

File: src/devices/Ws28xx/Ws28xx.cs [hunk 2]
```diff
@@ -2,22 +2,22 @@
...
-        private readonly SpiDevice _spiDevice;
-        public BitmapImageNeo3 Image { get; }
+        protected readonly SpiDevice _spiDevice;
+        public BitmapImage Image { get; protected set; }
 
-        public Ws2812b(SpiDevice spiDevice, int width, int height = 1)
+        public Ws28xx(SpiDevice spiDevice, int width, int height = 1)
         {
             _spiDevice = spiDevice;
             _spiDevice.ConnectionSettings.ClockFrequency = 2_400_000;
```
Comment (krwq) [inline src/devices/Ws28xx/Ws28xx.cs]: I know this isn't you but this seems incorrect that we override the SPI settings - must have missed it in the other review

File: src/devices/Ws28xx/BitmapImageWs2808.cs (hunk 1 above)
Comment (DemoS76) [reply src/devices/Ws28xx/BitmapImageWs2808.cs]: Fixed

File: src/devices/Ws28xx/Ws28xx.cs (hunk 2 above)
Comment (DemoS76) [reply src/devices/Ws28xx/Ws28xx.cs]: Fixed

File: src/devices/Ws28xx/Ws2808.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Spi;
+
+namespace Iot.Device.Ws28xx
+{
+    public class Ws2808 : Ws28xx
+    {
+        public Ws2808(SpiDevice spiDevice, int width, int height = 1)
+            :base(spiDevice, width, height)
```
Comment (krwq) [inline src/devices/Ws28xx/Ws2808.cs]: nit: space after `:`

File: src/devices/Ws28xx/Ws2812B.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Spi;
+
+namespace Iot.Device.Ws28xx
+{
+    public class Ws2812b : Ws28xx
+    {
+        public Ws2812b(SpiDevice spiDevice, int width, int height = 1)
+            :base(spiDevice, width, height)
```
Comment (krwq) [inline src/devices/Ws28xx/Ws2812B.cs]: ditto

**Pull request merged**